test/data/composer/osbuild-composer.toml

```diff
@@ -1,7 +1,9 @@
 [koji]
-allowed_domains = [ "localhost", "client.osbuild.org" ]
+allowed_domains = [ "localhost", "client.osbuild.org", "composer", "worker", "worker.osbuild.org", "composer.osbuild.org" ]
 ca = "/etc/osbuild-composer/ca-crt.pem"
 
 [worker]
-allowed_domains = [ "localhost", "worker.osbuild.org" ]
+allowed_domains = [ "localhost", "worker" ]
 ca = "/etc/osbuild-composer/ca-crt.pem"
+
```

Comment:
Was this changed for your local testing or do we need it for the changes here?

Reply:
I'll revert this, that's a mistake

Reply:
Seems you missed this? :)

Reply:
I think this is the only thing left :)